15-Appendix.Rmd

```diff
@@ -0,0 +1,144 @@
+# (APPENDIX) Appendix {-}
+
+# Conda Package Management
+
+## Unified package management for both R and Python
+
+### Faster builds
+
+CRAN[^what_is_cran] mirrors do not provide linux[^linux_note] compiled binaries
+for packages. This means a long wait when doing `install.packages` both in an R
+Studio session and when running a Docker build for a `shiny` application.
+
+### Cross Language
+
+Both `dbtools` and `s3tools` rely on Python packages through the `reticulate` R
+to Python bridge. This means that `packrat` is not enough to reproducibly and
+reliably manage all of your application's dependencies.
+
+`conda` supports managing both Python and R dependencies in a single
+`environment`. It can make sure all of these libraries are compatible with each
+other.
+
+## Installing Packages
+
+If you need to find a package name you can use the [anaconda search] to find the
+name. Run `conda install PACKAGENAME` in the `Terminal` tab to install it.
+
+### R
+
+Most CRAN (around 95%) are available through `conda`, they retain the same name
```

Comment:
I think this should read
`Around 95% CRAN packages are available through `conda`, and have the same name with an additional `r-` prefix (e.g. `Rcpp` becomes `r-Rcpp`).